src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/rc_penfing_resumption_handler_test.cc

```diff
@@ -0,0 +1,425 @@
+/*
```

Comment:
typo in file name: `rc_penfing_resumption_handler_test.cc` -> `rc_pending_resumption_handler_test.cc`

Reply:
@jacobkeeler 
Fixed in:https://github.com/smartdevicelink/sdl_core/pull/3480/commits/5ecc652987d9f3871c656d9e030ea2dab11deef4